docs/ml/vision/_index.md

```diff
@@ -18,23 +22,97 @@ While the camera component lets you access what your machine's camera sees, the
+- `class_name` (string): specifies the label of the found object.
+- `confidence` (float): specifies the confidence of the assigned label.
+  Between `0.0` and `1.0`, inclusive.
+
+**Supported API methods:**
+
+- [GetDetections()](#getdetections)
+- [GetDetectionsFromCamera()](#getdetectionsfromcamera)
+
+## Classifications
+
+_2D Image Classification_ is the process of taking a 2D image from a camera and deciding which class label, out of many, best describes the given image.
+Any camera that can return 2D images can use 2D image classification.
+
+The class labels used for classification vary and depend on the machine learning model and how it was trained.
+
+The returned classifications consist of the image's class label and confidence score.
+
+- `class_name` (string): specifies the label of the found object.
+- `confidence` (float): specifies the confidence of the assigned label.
+  Between `0.0` and `1.0`, inclusive.
+
+**Supported API methods:**
+
+- [GetClassifications()](#getclassifications)
+- [GetClassificationsFromCamera()](#getclassificationsfromcamera)
+
+## Segmentations
+
+_3D Object Segmentation_ is the process of separating and returning a list of the identified "objects" from a 3D scene.
+The "objects" are usually a list of point clouds with associated metadata, like the label, the 3D bounding box, and center coordinates of the object.
+
+3D object segmentation is useful for obstacle detection.
+See our guide [Navigate with a Rover Base](/tutorials/services/navigate-with-rover-base/#next-steps-automate-obstacle-detection) for an example of automating obstacle avoidance with 3D object segmentation for obstacle detection.
+
+Any camera that can return 3D pointclouds can use 3D object segmentation.
+
+**Supported API methods:**
+
+- [GetObjectPointClouds()](#getobjectpointclouds)
+
+## Supported Models
+
+To use your motor with Viam, check whether one of the following [built-in models](#built-in-models) or [modular resources](#modular-resources) supports your motor.
+
+{{< readfile "/static/include/create-your-own-mr.md" >}}
+
+### Built-in models
+
+For configuration information, click on the model name:
+
+<!-- prettier-ignore -->
+Model | Description <a name="model-table"></a>
+----- | -----------
+[`mlmodel`](./mlmodel/) | A detector or classifier that uses a `tensorflow-lite` model available on the machine’s hard drive to draw bounding boxes around objects or returns a class label and confidence score.
+[`color_detector`](./color_detector/) | A heuristic detector that draws boxes around objects according to their hue (does not detect black, gray, and white).
+[`obstacles_pointcloud`](./obstacles_pointcloud/) | A segmenter that identifies well-separated objects above a flat plane.
+[`detector_3d_segmenter`](./detector_3d_segmenter/) | This model takes 2D bounding boxes from an object detector and projects the pixels in the bounding box to points in 3D space.
```

Comment:
```suggestion
[`detector_3d_segmenter`](./detector_3d_segmenter/) | A segmenter that takes 2D bounding boxes from an object detector and projects the pixels in the bounding box to points in 3D space.
```
(opt) If that's the case maybe this could help clear up if others were similarly confused?

Reply:
Perfect yep